### PR TITLE
Chudin_Pavlo 

### PR DESCRIPTION
**Pull request by Pashahu1**
[DEMO LINK](https://Pashahu1.github.io/layout__catalog/)
[TEST REPORT LINK](https://Pashahu1.github.io/layout__catalog/report/html_report/)

### CODE REVIEW

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
Fix demo and test report link

If you don't understand something or need some help, please, ask it in fe_chat and you will get answers to all questions

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,337 @@
...
+            <a class="header__nav-link" href="#Smartphones">Smartphones</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#Laptops & Computers"
+              data-qa="nav-hover"
+              >Laptops & Computers
+            </a>
```
Comment (vlad-demchuk) [inline src/index.html:36]:
```suggestion
            >
                Laptops & Computers
            </a>
```

File: src/index.html [hunk 2]
```diff
@@ -5,8 +5,337 @@
...
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="cards__card-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card-cost">
```
Comment (vlad-demchuk) [inline src/index.html:81]: no need to create a separate bem block here

File: src/reseat/reseat.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+* {
```
Comment (vlad-demchuk) [inline src/reseat/reseat.scss:1]: typo in the file name

reset.css

File: src/reseat/reseat.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  list-style: none;
+}
+
+h2,
+p,
+body,
+html {
+  font-family: "Roboto", sans-serif;
+  padding: 0;
+  margin: 0;
```
Comment (vlad-demchuk) [inline src/reseat/reseat.scss:16]: html doesn't have paddings or margins

specify elements you need to reset

File: src/styles/blocks/cards.scss [hunk 5]
```diff
@@ -0,0 +1,101 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  margin: 0 auto;
```
Comment (vlad-demchuk) [inline src/styles/blocks/cards.scss:6]: image

File: src/styles/blocks/cards.scss [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    background: $white-color;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+
+    padding: 32px 16px 16px;
+    width: 200px;
+    height: 406px;
+  }
+
+  .page-catalog {
```
Comment (vlad-demchuk) [inline src/styles/blocks/cards.scss:25]: no need to create a separate block here

File: src/styles/blocks/cards.scss [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+    flex-wrap: wrap;
+    justify-content: center;
+    @include cards-gap;
+    background: $white-color;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+
+    padding: 32px 16px 16px;
+    width: 200px;
+    height: 406px;
```
Comment (vlad-demchuk) [inline src/styles/blocks/cards.scss:22]: do not hardcode height

let card content dictate this value

File: src/styles/blocks/cards.scss [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__card-title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $blue-dark-color;
```
Comment (vlad-demchuk) [inline src/styles/blocks/cards.scss:39]: css properties order
http://codeguide.academy/html-css.html#css-order

File: src/styles/blocks/cards.scss [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+    color: $white-color;
+
+    background-color: $arial-color;
+    border: 1px solid $arial-color;
+    border-radius: 5px;
+  }
+
+  &__card-link:hover {
+    background-color: $white-color;
+    color: $arial-color;
```
Comment (vlad-demchuk) [inline src/styles/blocks/cards.scss:99]: you can use nesting for hover in the class above

```
&__card-link {
some styles

  &:hover {
     background-color: $white-color;
     color: $arial-color;
  }
}

File: src/styles/blocks/header.scss [hunk 10]
```diff
@@ -0,0 +1,78 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+
+  padding: 0 50px;
+  min-width: 0;
```
Comment (vlad-demchuk) [inline src/styles/blocks/header.scss:6]: why do you need `min-width: 0;`?

File: src/styles/blocks/header.scss [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+  max-width: 100%;
+  height: 60px;
```
Comment (vlad-demchuk) [inline src/styles/blocks/header.scss:8]: do not hardcode height

File: src/styles/blocks/header.scss [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+    line-height: 14px;
+    text-transform: uppercase;
+    list-style: none;
+  }
+
+  &__nav-item {
+    margin-right: 20px;
+  }
+
+  &__nav-item:last-of-type {
```
Comment (vlad-demchuk) [inline src/styles/blocks/header.scss:40]: can we use `last-child` or `not` for `&__nav-item`?

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 13]
```diff
@@ -5,8 +5,338 @@
...
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#Video">Video</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards">
+      <article class="cards__card" data-qa="card">
```
Comment (vlad-demchuk) [inline src/index.html:61]: also, let's create a separate bem block for card, it will make our code more reusable

File: src/reseat/reset.css [hunk 14]
```diff
@@ -0,0 +1,16 @@
...
+ul {
+  padding: 0;
+  list-style: none;
+}
+
+h2,
+p,
+body {
+  font-family: "Roboto", sans-serif;
+  padding: 0;
```
Comment (vlad-demchuk) [inline src/reseat/reset.css:14]: body doesn't have paddings

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
Looks better🙂
let's improve your code a bit

center the catalog
image

**Review by volodymyr-soltys97 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**